src/app/search/items/search-filters/sockets.ts

```diff
@@ -337,6 +338,35 @@ const socketFilters: ItemFilterDefinition[] = [
       return (_item) => false;
     },
   },
+  {
+    keywords: 'readytoenhance',
```

Comment:
keyword name up for debate; enhanceableperks vs this vs something else

Reply:
I think I like `enhanceableperks` a little better.

Reply:
Taking a look at renaming. Also considering what `not:enhanceableperks` would mean vs `not:readytoenhance`

Reply:
✋ one vote here for `enhancementready`